src/sass/app/trade.scss

```diff
@@ -528,8 +530,7 @@
                 padding: 0 10px;
 
                 > p {
-                    margin: auto 0 5px;
-                    padding-top: 10px;
+                    margin: auto 0 0;
```

Comment:
any reason for no margin here? if we remove all styles from this paragraph then the jump disappears :)

Reply:
@easteregg please check this one out, once you are free :)

Reply:
@ruskakimov Will do sir.

Reply:
Done. you are right, I missed the jumping results already 😂